neuro_san/interfaces/agent_name_mapper.py

```diff
@@ -0,0 +1,32 @@
+
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+
```

Comment:
Interface for abstract agent mapping policy.

Reply:
Interesting. Makes me wonder if we could use this for separating out mcp vs neuro-san vs a2a agent namespaces as well. (Thoughts?)

Reply:
Hmm. Do we need separate namespaces? I thought the idea is that we could provide the same functionality (set of agent networks) presented by different protocols (MCP, A2A, whatever).
That said though - in general yes. When we enable some protocol, we can specify any agent -> agent definition file mapping as necessary. Maybe with different registry roots? Or not.